### ISSUE TITLE
Fix snapshot testing

### ISSUE THREAD

**Issue by j-rewerts** [yeg-relief/youcanbenefit#74]
We currently have problems with reproducibility. Our snapshots often change:
``` bash
      Object {
        "guid": "hp5mupqcVY8ZMiKg7Q91Uoi4Wf",
        "queryIDs": Array [
    -     "C8NZoL1Nf7VEgEC4VMFI9GlebN",
          "d5NswIlUvzxY0AJ1hGm4eKQtS4",
    +     "fMwdx1jiAZUt09ALDer98GardT",
          "JeXk8cOZnqJMepmHpxDqpsEXNE",
          "jI1HlTlpbULsSVUmHnYi3ZlVJ7",
          "jzMQkEK41gnhmwWyxAuWF3eZ4d",
          "NOyR66aE30dJYXgdHJ2CvGLpnx",
          "qZyokzKnpIw3Ue6mdTi8je3knK",
```

We need our snapshots to be the exact same between runs.

**Comment by CodyGramlich** [comment]
This seems to be only a problem with GET /protected/program/:guid. It only returns 10 queries when there are 11 queries for this program. The search size for elasticsearch wasn't set on EsQuery.service.ts getByGuid(), so I changed it to a size of 10000. Also, I don't think this endpoint is ever used in the application currently.

**Comment by j-rewerts** [comment]
Fixed by @CodyGramlich with #87.
